app/helpers/bootstrap_form_builder.rb

```diff
@@ -0,0 +1,34 @@
+def errors_block(attribute, feedback_name)
```

Comment:
Nope! Tā nevajag darit! Ruby to arļauj daŗit un Javascript, bet ši metode tiks izmantota tikai šaja klasē, tāpēc lūdzu iznes to iekš klases zem blokā kur tu overraido metodes.